backends/ref/ceed-ref-restriction.c

```diff
@@ -373,19 +458,15 @@ int CeedElemRestrictionCreate_Ref(CeedMemType mem_type, CeedCopyMode copy_mode,
     case 381:
       impl->Apply = CeedElemRestrictionApply_Ref_381;
       break;
-    // LCOV_EXCL_START
```

Comment:
oops, those exclusion statements can probably be combined but should be there

Reply:
Ah yes, I meant to ask about this. I was confused why certain cases were fine without the exclusions and others were not. Is it just to simplify coverage and exclude some sizes?

Reply:
Right, there are a few cases that list that are helpful for work in other places but don't want to test here. But that 'uncovered' code just adds noise to the code coverage reports

Reply:
Sounds good, I'll add them back